pycbc/types/timeseries.py

```diff
@@ -101,14 +107,12 @@ def _typecheck(self, other):
         if isinstance(other, TimeSeries):
             if not self.sample_rate_close(other):
                 raise ValueError('different delta_t')
-            if self._epoch != other._epoch:
+            if not self.epoch_close(other):
                 raise ValueError('different epoch')
```

Comment:
While you are at it, I suggest
```python
raise ValueError('different epoch, {} vs {}'.format(self.start_time, other.start_time))
```

Reply:
Ok, suggestion taken.

Reply:
Also changed the different detla_t message while I was at it.